tests/framework/PostProcessors/Metrics/gold/mcMetricRun/output_metric.csv

```diff
@@ -0,0 +1,5 @@
+Metrics,ans-ans2
+euclidean2_paired_distance_euclidean,8.15320610438
```

Comment:
it is not standard

Reply:
removed, this is not needed, and this is the old output format.